### DISCUSSION TITLE
[createReceiptShipment] : What's the field identifier that shipment notification has already been sent?

### DISCUSSION THREAD

**Discussion by joyceeeen** [etsy/open-api#876] [Q&A] [answered]
In V2, whenever a shipment notification has already been sent, when you call createReceiptShipment, the API is returning this error **"Shipping notification email has already been sent for this receipt."**, but now we are not getting this. Whats the alternative of this in V3?

**Comment by Gareth064** [comment, marked as answer]
You can check a Receipt to see if it has already been shipped with a shipping notification etc.

![image](https://user-images.githubusercontent.com/22307223/221245374-a2cfd2fe-8ecf-4466-8e72-6c10f9b9f824.png)
